src/i18n/README.md

```diff
@@ -0,0 +1,122 @@
+# i18n setup
```

Comment:
Maybe instead just link the README from Yoroi-frontend? I'm concerned about these getting out of sync

Reply:
Indeed, compared both files and they are essentially the same. Link added.